src/components/frame/Extensions/pages/ColonyDetailsPage/ColonyDetailsPage.tsx

```diff
@@ -16,85 +13,80 @@ const displayName = 'frame.Extensions.pages.ColonyDetailsPage';
 const ColonyDetailsPage: FC = () => {
   const { formatMessage } = useIntl();
   const isMobile = useMobile();
-  const { colony, loading } = useColonyContext();
+  const { colony } = useColonyContext();
   const { name, metadata } = colony || {};
   const { avatar, thumbnail, description, externalLinks } = metadata || {};
 
   return (
-    <Spinner
-      loading={loading}
-      loadingText={{ id: 'loading.colonyDetailsPage' }}
-    >
-      <TwoColumns aside={<Navigation pageName="extensions" />}>
-        <div className={clsx('pt-[4.375rem] px-6 pb-6 mt-10', styles.box)}>
-          <div className="absolute left-6 -top-11 rounded-full border-4 border-gray-100 flex">
-            <Avatar size="md" avatar={avatar || thumbnail || ''} />
-          </div>
-          <h2 className="heading-2 mb-1">{name}</h2>
-          <p className="text-md text-gray-600 mb-5">{description}</p>
-          <div className="mb-6 flex gap-x-2">
-            {externalLinks?.length ? (
-              externalLinks.map(({ name: linkName, link }) => (
-                <a key={`${linkName}:${link}`} href={link}>
-                  {linkName}
-                </a>
-              ))
-            ) : (
-              <TextButton
-                iconName="plus"
-                iconSize="tiny"
-                text={{ id: 'button.socialLinks' }}
-                mode="medium"
-              />
-            )}
-          </div>
-          {/* @TODO: Add functionality to edit colony details */}
-          <Button
-            mode="primarySolid"
-            text={{ id: 'button.editColonyDetails' }}
-            isFullSize={isMobile}
-          />
+    <div>
+      <div className={clsx('pt-[4.375rem] px-6 pb-6 mt-10', styles.box)}>
+        <div className="absolute left-6 -top-11 rounded-full border-4 border-gray-100 flex">
+          <Avatar size="md" avatar={avatar || thumbnail || ''} />
         </div>
-        <div
-          className={clsx(
-            'p-6 mt-12 flex flex-col justify-between items-center gap-6 sm:flex-row sm:gap-12',
-            styles.box,
+        <h2 className="heading-2 mb-1">{name}</h2>
+        <p className="text-md text-gray-600 mb-5">{description}</p>
+        <div className="mb-6 flex gap-x-2">
+          {externalLinks?.length ? (
+            externalLinks.map(({ name: linkName, link }) => (
+              <a key={`${linkName}:${link}`} href={link}>
+                {linkName}
+              </a>
+            ))
```

Comment:
We have a component called `ExternalLink` for... external links 😎 👉 👉  I know it was implemented like this already before you touched it (By PagePro perhaps?) but I'm just mentioning it for future reference

Reply:
Just copied it. But will change.